src/components/Projects/MyProjects.tsx

```diff
@@ -0,0 +1,85 @@
+import { t, Trans } from '@lingui/macro'
+import Grid from 'components/shared/Grid'
+import Loading from 'components/shared/Loading'
+import ProjectCard from 'components/shared/ProjectCard'
+import { NetworkContext } from 'contexts/networkContext'
+import { ThemeContext } from 'contexts/themeContext'
+import { useMyProjectsQuery } from 'hooks/v1/Projects'
+import React, { useContext } from 'react'
+import { InfoCircleOutlined } from '@ant-design/icons'
+
+export default function MyProjects() {
+  const { userAddress } = useContext(NetworkContext)
+
+  const { data: projects, isLoading } = useMyProjectsQuery(userAddress)
+
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+
+  return (
+    <React.Fragment>
+      {!userAddress ? (
+        <div
+          style={{
+            textAlign: 'center',
+            color: colors.text.disabled,
+            padding: 20,
+          }}
+          hidden={isLoading}
+        >
+          <Trans>Connect your wallet to see your projects.</Trans>
+        </div>
+      ) : (
+        <React.Fragment>
+          {projects && projects.length > 0 && (
+            <React.Fragment>
+              <Grid>
+                {projects.map(p => (
+                  <ProjectCard project={p} />
+                ))}
+              </Grid>
+            </React.Fragment>
+          )}
+
+          {!isLoading &&
+            projects &&
+            (projects.length === 0 ? (
+              <div
+                style={{
+                  textAlign: 'center',
+                  color: colors.text.disabled,
+                  padding: 20,
+                }}
+                hidden={isLoading}
+              >
+                <Trans>You haven't created any projects yet.</Trans>
+              </div>
+            ) : (
+              <div
+                style={{
+                  textAlign: 'center',
+                  color: colors.text.disabled,
+                  padding: 20,
+                }}
+              >
+                {projects.length}{' '}
+                {projects.length === 1 ? t`project` : t`projects`}{' '}
+              </div>
+            ))}
+
+          {isLoading && (
+            <div style={{ marginTop: 40 }}>
+              <Loading />
+            </div>
+          )}
+
+          <p style={{ marginBottom: 40, marginTop: 40, maxWidth: 800 }}>
```

Comment:
**suggestion** could we only show this when there are some projects? Other than that, looks great 👏

Reply:
By 'this', do you mean the tab itself? The weird thing is that fetching projects/holdings is an asynchronous thing, so we would either have to display the tabs by default and remove them if none are found, or add them to the UI after the query returns what it has found.

Reply:
@jmilldotdev Oh no! Sorry, should have been more clear.

I meant this "Projects that you have created." message. Maybe we can only show this message when there are > 0 projects. WDYT?

Reply:
Done, also did the same for the holdings tab